### ISSUE TITLE
`disconnect` function from `useConnectors` only disconnect in the current "session" but not after a refresh

### ISSUE THREAD

**Issue by YohanTz** [apibara/starknet-react#169]

**Comment by fracek** [comment]
I believe this is related to #170, need to track the user preferences on top of wallet availability.

**Comment by YohanTz** [comment]
Yes totally, I can give some help if needed, as mentioned in #170 🙂 @fracek

**Comment by fracek** [comment]
Of course! I would wait for #171 to be merged (eta today) before start coding. Can you share a bit how you plan to add it? IIRC wagmi does something similar and they just use local storage.

**Comment by YohanTz** [comment]
Ok, great ! I planned to simply store the last connected wallet id in the local storage and nothing if the wallet has been disconnected. This way, we can check on page load to which wallet we should try to connect

**Comment by YohanTz** [comment]
Hey @fracek ! I just started to work on this issue, I just have a question about it, do we want this behaviour to be under the autoConnect option or under a new option that we can add ?

**Comment by fracek** [comment]
Fixed in #186.

**Comment by fracek** [comment]
> Hey @fracek ! I just started to work on this issue, I just have a question about it, do we want this behaviour to be under the autoConnect option or under a new option that we can add ?

They way you implemented it is great!
